### ISSUE TITLE
zyBooks: text input has wrong color

### ISSUE THREAD

**Issue by AmityWilder** [HenryWilder/amitygxmod#9]
Certain text inputs in zyBooks have black text, which is very difficult to read against dark-scheme backgrounds. This is of high priority due to the poor accessibility.
![image](https://github.com/HenryWilder/amitygxmod/assets/74995093/a362e6e4-aabb-4f1d-8946-4053dd7e31f9)

The associated HTML:
```html
<div class="question-container ">
  <div class="input">
    <div class="highlight">
      <pre>
        <textarea id="ember..." class="ember-text-area ember-view zb-text-area hide-scrollbar" aria-labelledby="..." autocapitalize="off" cols="20" rows="1" spellcheck="false"></textarea>
      </pre>
    </div>
    <!-- (unrelated) -->
  </div>
</div>
```
